microsoft-edge/devtools-guide-chromium/dom/index.md

```diff
@@ -22,70 +22,70 @@ ms.date: 07/22/2022
    limitations under the License.  -->
 # Get started viewing and changing the DOM
 
-Follow these interactive tutorial sections to learn the basics of viewing and changing the [Document Object Model](https://developer.mozilla.org/docs/Web/API/Document_Object_Model) (DOM) of a page using Microsoft Edge DevTools.
+To learn the basics of viewing and changing the [Document Object Model](https://developer.mozilla.org/docs/Web/API/Document_Object_Model) (DOM) of a page using Microsoft Edge DevTools, follow these interactive tutorial sections.
 
 To understand the difference between the DOM and HTML, see [Appendix: HTML versus the DOM](#appendix-html-versus-the-dom), below.
 
 
 <!-- ====================================================================== -->
 ## View DOM nodes
 
-The DOM Tree of the Elements panel is where you do all DOM-related activities in DevTools.
+The DOM Tree of the **Elements** tool is where you do all DOM-related activities in DevTools.
 
 
-### Inspect a node
+#### Inspect a node
 
 When you are interested in a particular DOM node, **Inspect** is a fast way to open DevTools and investigate that node.
 
 1. Open the [DOM Examples](https://microsoftedge.github.io/Demos/devtools-dom-get-started/) demo page in a new window or tab.  To do this, right-click the link, or press and hold **Ctrl** (Windows, Linux) or **Command** (macOS) and then click the link.
```

Comment:
```suggestion
1. Open the [DOM Examples](https://microsoftedge.github.io/Demos/devtools-dom-get-started/) demo page in a new window or tab.
```
"the link" is ambiguous - the link in previous sentence, or some link in Demos nav page?
We can probably assume user can figure out to right-click the provided link and examine the menuitems.  Instead of mixing-in basics of how to click a link, into these instructions.